02-into_to_imaging.Rmd

```diff
@@ -0,0 +1,17 @@
+
+# Introduction to Imaging
+
+
+## Learning Objectives
+
+```{r, fig.alt = "", out.width = "100%", echo = FALSE}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1ME0NbcIBmnHJRhX3JJyCwJuuomkl_BjJp6lD5oD5WnU/edit#slide=id.gd422c5de97_0_10")
```

Comment:
```suggestion
ottrpal::include_slide("https://docs.google.com/presentation/d/1ME0NbcIBmnHJRhX3JJyCwJuuomkl_BjJp6lD5oD5WnU/edit#slide=id.g273f37f3566_2_0")
```

Reply:
Added a new slide in the slide deck that is a placeholder/specifies info needs to be filled in